com.unity.formats.fbx/Editor/Unity.Formats.Fbx.Editor.api

```diff
@@ -0,0 +1,125 @@
+    {
+        public virtual bool AllowSceneModification { get; }
+        public virtual bool EmbedTextures { get; }
+        public virtual bool ExportUnrendered { get; }
+        public virtual bool KeepInstances { get; }
+        public virtual UnityEditor.Formats.Fbx.Exporter.LODExportType LODExportType { get; }
+        public virtual UnityEditor.Formats.Fbx.Exporter.Include ModelAnimIncludeOption { get; }
+        public virtual UnityEditor.Formats.Fbx.Exporter.ObjectPosition ObjectPosition { get; }
+        public virtual bool PreserveImportSettings { get; }
+        public ExportModelSettingsSerialize() {}
+        public virtual bool Equals(object e);
+        public virtual int GetHashCode();
+        public void SetEmbedTextures(bool embedTextures);
+        public void SetExportUnredererd(bool exportUnrendered);
+        public void SetKeepInstances(bool keepInstances);
+        public void SetLODExportType(UnityEditor.Formats.Fbx.Exporter.LODExportType lodLevel);
+        public void SetModelAnimIncludeOption(UnityEditor.Formats.Fbx.Exporter.Include include);
+        public void SetObjectPosition(UnityEditor.Formats.Fbx.Exporter.ObjectPosition objectPosition);
+        public void SetPreserveImportSettings(bool preserveImportSettings);
+    }
+
+    public abstract class ExportOptionsSettingsSerializeBase : UnityEditor.Formats.Fbx.Exporter.IExportOptions
+    {
+        public abstract bool AllowSceneModification { get; }
+        public virtual bool AnimateSkinnedMesh { get; }
+        public virtual UnityEngine.Transform AnimationDest { get; }
+        public virtual UnityEngine.Transform AnimationSource { get; }
+        public virtual bool EmbedTextures { get; }
+        public virtual UnityEditor.Formats.Fbx.Exporter.ExportFormat ExportFormat { get; }
+        public abstract bool ExportUnrendered { get; }
+        public virtual bool KeepInstances { get; }
+        public abstract UnityEditor.Formats.Fbx.Exporter.LODExportType LODExportType { get; }
+        public abstract UnityEditor.Formats.Fbx.Exporter.Include ModelAnimIncludeOption { get; }
+        public abstract UnityEditor.Formats.Fbx.Exporter.ObjectPosition ObjectPosition { get; }
+        public virtual bool PreserveImportSettings { get; }
+        public virtual bool UseMayaCompatibleNames { get; }
+        protected ExportOptionsSettingsSerializeBase() {}
+        public virtual bool Equals(object e);
+        public virtual int GetHashCode();
+        public void SetAnimatedSkinnedMesh(bool animatedSkinnedMesh);
+        public void SetAnimationDest(UnityEngine.Transform dest);
+        public void SetAnimationSource(UnityEngine.Transform source);
+        public void SetExportFormat(UnityEditor.Formats.Fbx.Exporter.ExportFormat format);
+        public void SetUseMayaCompatibleNames(bool useMayaCompNames);
+    }
+
+    public interface IExportOptions
```

Comment:
can you make the interface internal and have a "hidden implementation" ?

Reply:
possibly, I'll see what I can do